src/character.h

```diff
@@ -987,6 +987,11 @@ class Character : public Creature, public visitable
         /** Add or removes a mutation on the player, but does not trigger mutation loss/gain effects. */
         void set_mutations( const std::vector<trait_id> &traits );
         void set_mutation( const trait_id & );
+        // Set a mutation, but don't do any of the necessary updates
+        // Only call this from one of the above two functions
+        void set_mutation_unsafe( const trait_id & );
```

Comment:
I admit I'm not very familar with Cata's code standards, but is there any reason not to make this method private?  It seems like a potential foot-gun and locking it down couldn't hurt in my opinion.

Reply:
It definitely should be private, I just didn't think of it.